kousa/priv/repo/migrations/20210508165321_staff.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Beef.Repo.Migrations.Staff do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :staff, :boolean, default: false
```

Comment:
its fine to create 2 migrations, but you can also do both fields in a single migration

Reply:
done